E2E/pageObjects/game.page.ts

```diff
@@ -31,6 +32,14 @@ export = {
     I.dontSeeElement(locator.chessPieces.container);
     I.seeElement(locator.buttons.copyLink);
     I.seeElement(locator.buttons.restartGame);
-  }
+  },
 
+  checkCopyLinkButton: () => {
+    I.waitForElement(locator.buttons.copyLink, 10);
+    I.click(locator.buttons.copyLink);
+    I.waitForElement(locator.text.linkCopiedToClipboard, 7);
```

Comment:
Why are you using different explicit timeouts here? (10 and 7). I would expect default codecept timeouts to be enough.